exchanges/orderbook/incremental_updates.go

```diff
@@ -97,17 +97,21 @@ func (d *Depth) ProcessUpdate(u *Update) error {
 		}
 	}
 
+	if !d.verifyOrderbook {
```

Comment:
🚧  I think since we're here and touching things inappropriately, we might as well go full-catholic
Might need to consider overlap of field name and possible method name, but since it's not a method right now that's okay.
```suggestion
	if !d.validateOrderbook {
```